payments/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+  mongo:
+    container_name: mongo
+    image: mongo
+    restart: always
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=${MONGO_ROOT_USER}
+      - MONGO_INITDB_ROOT_PASSWORD=${MONGO_ROOT_PASSWORD}
+      - MONGO_INITDB_DATABASE=project
```

Comment:
The database name should be same as the database name in `.env` file

Reply:
@nicamanas you might wna also add a `.env.example` with mocked values to show how the .env should look like in order to init this app/MS successfully

Reply:
@zhiweit negative on that, the database name in the .env file is within the mongoDB instance.